docs/spectral_regions.rst

```diff
@@ -209,6 +209,53 @@ An example of a multiple sub-region `~specutils.SpectralRegion`:
     >>> sub_spectra[1].spectral_axis
     <SpectralAxis [34., 35., 36., 37., 38., 39., 40.] nm>
 
+
+Line List Compatibility
```

Comment:
Neat!  A good change, but maybe it should be called out in the changelog as a new feature since it's otherwise buried in this PR with several other things?

Reply:
Yeah, there are several that I have listed that will go into the change log, but it's been happening usually as a PR before the release.

Reply:
I didn't realize that's when you were populating it - we should definitely move away from that - it's dangerous to rely on memory, etc to update the changelog especially with more people potentially doing releases in the future.

But so as not to delay this PR any further I'll do this myself right now!